### PR TITLE
Test company view

### PR DESCRIPTION
**Pull request by Khalid-Abed**

### CODE REVIEW

**Review by Ahmed-Elhadary — DISMISSED** [review]

File: app/Http/Requests/JobRequest.php [hunk 1]
```diff
@@ -24,21 +24,20 @@ public function authorize()
...
-            'min_years_of_experience' =>'required|numeric',
-            'max_years_of_experience' => 'required|numeric',
-            'vacancies' =>'required|numeric' ,
-            'min_salary' => 'required|numeric',
-            'max_salary' => 'required|numeric',
-            'description' => 'required|string',
-            'requirements' => 'required|string',
+            'city_id' =>'required',
+            'min_years_of_experience' =>'required|numeric|integer|min:0',
+            'max_years_of_experience' => 'required|numeric|integer|min:0',
```
Comment (ahmed-fawzy-metwally) [inline app/Http/Requests/JobRequest.php]: Relplace ```'max_years_of_experience' => 'required|numeric|integer|min:0',```
with ```'max_years_of_experience' => 'required|numeric|integer|min:1'```

File: app/Http/Requests/JobRequest.php [hunk 2]
```diff
@@ -24,21 +24,20 @@ public function authorize()
...
+            'vacancies' =>'required|numeric|integer|min:0' ,
+            'min_salary' => 'required|numeric|integer|min:0',
+            'max_salary' => 'required|numeric|integer|min:0',
```
Comment (ahmed-fawzy-metwally) [inline app/Http/Requests/JobRequest.php]: Replace ```'vacancies' =>'required|numeric|integer|min:0'```
With ```'vacancies' =>'required|numeric|integer|min:1'```
Replace ```'min_salary' => 'required|numeric|integer|min:0'```
With ```'min_salary' => 'required|numeric|integer|min:1500'```
Replace ```'required|numeric|integer|min:0'```
With ```''required|numeric|integer|min:15001'```

File: resources/views/company/jobs/show.blade.php [hunk 3]
```diff
@@ -54,16 +53,16 @@
                    </div>
                 </div>
 
-                <div class="row p-5 bg-white my-5 bord">
+                <div class="row p-5 bg-white my-3 shadow-element bord">
                     <div class="">
-                        <h3 class="underline">Job Categories:</h3>
+                        <h3 class="">Job Categories:</h3>
```
Comment (ahmed-fawzy-metwally) [inline resources/views/company/jobs/show.blade.php:58]: Remove ```class=""```

File: resources/views/company/jobs/show.blade.php [hunk 4]
```diff
@@ -74,16 +73,16 @@
                     </div>
                 </div>
 
-                <div class="row p-5 bg-white my-5 bord">
+                <div class="row p-5 shadow-element bg-white my-3 bord">
                     <div class="">
-                        <h3 class="underline">Job Requirements</h3>
+                        <h3 class="">Job Requirements</h3>
```
Comment (ahmed-fawzy-metwally) [inline resources/views/company/jobs/show.blade.php]: Remove ```class=""``` and the unuserfull parent ```div```

File: resources/views/company/jobs/show.blade.php [hunk 5]
```diff
@@ -54,16 +53,16 @@
...
                     <div class="">
-                        <h3 class="underline">Job Categories:</h3>
+                        <h3 class="">Job Categories:</h3>
                         <h5>{{$job->industryCategory->name}}</h5>
                     </div>
                 </div>
 
-                <div class="row p-5 bg-white my-5 bord">
+                <div class="row p-5 bg-white shadow-element my-3 bord">
                     <div class="">
```
Comment (ahmed-fawzy-metwally) [inline resources/views/company/jobs/show.blade.php]: Remove this ```div```

File: resources/views/company/jobs/edit.blade.php [hunk 6]
```diff
@@ -25,6 +25,25 @@
                     @enderror
                 </div>
 
+                <div class="form-group">
+                    <label class="" for="career_level_id"> Career Level</label>
```
Comment (ahmed-fawzy-metwally) [inline resources/views/company/jobs/edit.blade.php]: Remove ```Calss=""```

File: resources/views/company/jobs/edit.blade.php [hunk 7]
```diff
@@ -64,46 +83,6 @@
...
-                        @error('company_id')
-                        <li class="text-error">{{$message}}</li>
-                        @enderror
-                    </div>
-                </div>
-
-
                 <div class="form-row">
                     <div class="form-group col-md-6">
                         <label class="" for="country_id"> Country</label>
```
Comment (ahmed-fawzy-metwally) [inline resources/views/company/jobs/edit.blade.php]: Remove ```class=""```

File: resources/views/company/jobs/edit.blade.php [hunk 8]
```diff
@@ -186,7 +165,7 @@
 
                     <div class="form-group col-md-6">
                         <label class="" for="max_years_of_experience">Max Year Of Experience</label>
```
Comment (ahmed-fawzy-metwally) [inline resources/views/company/jobs/edit.blade.php]: Remove ```class=""```

File: resources/views/company/jobs/show.blade.php [hunk 9]
```diff
@@ -94,18 +93,27 @@
 
 
             <div class="col-lg-4 col-sm-12">{{--Parent Div => Right Div --}}
-                <div class="p-5 bg-white bord">
-                    <h3 class="underline">About {{$job->company->name}}</h3>
+                <div class="p-5 bg-white shadow-element bord">
+                    <h3 class="">About {{$job->company->name}}</h3>
                     <p class="">{{$job->company->about}}</p>
```
Comment (ahmed-fawzy-metwally) [inline resources/views/company/jobs/show.blade.php]: Remove these ```class=""```

File: app/Http/Requests/JobRequest.php (hunk 1 above)
Comment (Khalid-Abed) [reply app/Http/Requests/JobRequest.php]: done

File: app/Http/Requests/JobRequest.php (hunk 2 above)
Comment (Khalid-Abed) [reply app/Http/Requests/JobRequest.php]: done

File: resources/views/company/jobs/show.blade.php (hunk 4 above)
Comment (Khalid-Abed) [reply resources/views/company/jobs/show.blade.php]: done

File: resources/views/company/jobs/show.blade.php (hunk 9 above)
Comment (Khalid-Abed) [reply resources/views/company/jobs/show.blade.php]: all done

**Pull request closed without merging**